lua/dylib.lua

```diff
@@ -1,6 +1,80 @@
+module("dylib", package.seeall)
+
+package.loaded["gatekeeper"] = nil
+require "gatekeeper"
+
+--
+-- C functions exported through FFI
+--
+
+local ffi = require("ffi")
+
+dyc = ffi.cast("struct dynamic_config *", dy_conf_ptr)
```

Comment:
I just realized that if we export `get_dy_conf()` to Lua via FFI, we don't need this cast, neither adding `dy_conf_ptr` to the Lua state; simpler code is beautiful.